code/src/components/AddTask.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
```

Comment:
This is such a nice component. It looks super good both in code and on the page, very nicely done!